cmd/ddev/cmd/sequelpro.go

```diff
@@ -47,8 +47,8 @@ func handleSequelProCommand(appLocation string) (string, error) {
 		return "", err
 	}
 
-	if app.SiteStatus() != "running" {
-		return "", errors.New("app not running locally. Try `ddev start`")
+	if app.SiteStatus() != platform.SiteRunning {
+		return "", errors.New("App not running locally. Try `ddev start`")
```

Comment:
I'd say the error should be an error, not instructions to the user. So error would be something like "site should be running and is not"